src/AST/ASTIdentifier.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) ZALORA
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+import ASTExpression from './ASTExpression';
+import { Token } from '../Lexer';
```

Comment:
Strange that Token is in the Lexer. Can we move it to its own space?

Reply:
I have moved to `src/tokens.ts`, named the interface as `IToken` and renamed the interface as `Token` when importing. I wanted to have meaningful naming convention as i move the Token from Lexer.